lib/Doctrine/Cache/Memcache.php

```diff
@@ -105,13 +102,7 @@ protected function _doContains($id)
      */
     protected function _doSave($id, $data, $lifeTime = false)
     {
-        if ($this->_options['compression']) {
-            $flag = MEMCACHE_COMPRESSED;
-        } else {
-            $flag = 0;
-        }
-
-        return $this->_memcache->set($id, $data, $flag, $lifeTime);
+        return $this->_memcache->set($id, $data, $lifeTime);
     }
```

Comment:
Why is compression removed? You can use `$this->_memcache->setOption(Memcached::OPT_COMPRESSION, true)` in `__construct`